portal/templates/activity/generate-activity-report.njk

```diff
@@ -0,0 +1,61 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% extends "index.njk" %}
+
+{% block pageTitle %}
+  View activity logs
+{% endblock %}
+
+{% block content %}
+  {{ govukBackLink({
+    href: "/",
```

Comment:
Not entirely sure where the back link goes (and if this first page should even have one?)